app/Http/Controllers/Profile/ProfileController.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace App\Http\Controllers\Profile;
+
+use Kris\LaravelFormBuilder\FormBuilderTrait;
+use Illuminate\Http\Request;
+use App\Member;
+use App\Forms\UpdateProfileForm;
+use App\Http\Requests;
+use App\Http\Controllers\Controller;
+use Auth;
+use JsValidator;
+use Session;
+
+class ProfileController extends Controller
+{
+    use FormBuilderTrait;
+	
+    public function show()
+    {
+		$user = Auth::user();
```

Comment:
[nit] Indentation issues throughout the files: which editor are you using? might want to check your settings.